cfgmgr/sflowmgr.cpp

```diff
@@ -85,34 +70,42 @@ void SflowMgr::sflowUpdatePortInfo(Consumer &consumer)
                 new_port = true;
                 port_info.local_rate_cfg = false;
                 port_info.local_admin_cfg = false;
+                port_info.autoneg_enabled = false;
```

Comment:
I don't recommend checking autoneg in sflow code. Portsorch will update oper speed in both autoneg enabled and disabled scenarios. Please consider just using oper speed and reprogramming if the oper speed is different from configured speed

Reply:
Removed and handled